### PR TITLE
Version tags

### PR DESCRIPTION
**Pull request by harmatt**
add version_tag atribute to scoreset

### CODE REVIEW

**Review by EstelleDa — CHANGES_REQUESTED** [review]
Change type to variant format.

**Review by EstelleDa — COMMENTED** [review]
I think this one is fine.

File: dataset/models/scoreset.py [hunk 1]
```diff
@@ -206,18 +207,31 @@ class Meta:
...
+        verbose_name="Variant Format",
+    )
+
     # ---------------------------------------------------------------------- #
     #                       Methods
     # ---------------------------------------------------------------------- #
     @transaction.atomic
     def save(self, *args, **kwargs):
+        # all new uploads should have the new variant tag
+        self.variant_format = "MaveDB v2.x"
```
Comment (afrubin) [inline dataset/models/scoreset.py:222]: This should refer to MAVE-HGVS.

**Comment by afrubin** [comment]
This PR is missing a few things. Specifically, the version exists in the database but it's not displayed and the version can't be set via manage command (or any other method short of editing the database/Django admin).

The following features should be added:

- Variant version displayed on the score set page near where the license logo is.
- Variant version included in the header of downloaded .csv files.
- Variant version included in the score set metadata that's exported via the new variant API, as well as the regular Django API (although this probably happens automatically).
- Add a manage.py command `setdatatsetversion` that accepts a URN and a dataset version (as a string). See other custom manage.py commands for examples/inspiration.
- Migration files will need to be created before this can be merged.

**Pull request closed without merging**